build_extra/rust/rust.gni

```diff
@@ -151,3 +171,18 @@ template("rust_executable") {
     }
   }
 }
+
+template("rust_test") {
+  run_rustc(target_name) {
+    crate_name = target_name
+    crate_type = "bin"
+    testonly = true
```

Comment:
What's the difference with is_test?

Reply:
testonly is for checking that production targets do not depend on this.
is_test controls the behavior of rustc.

testonly could be used here to switch behavior, but it's not exactly what that variable is intended for. In the future we may want to have testonly components or something...